terraform/monitoring/00_provider.tf

```diff
@@ -27,7 +27,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">=3.23.0"
+      version = "~> 3.23.0"
```

Comment:
Please, align version constraint(s) in [terraform/01_provider.tf](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/62b12c8f74d7e9d6b242fe1a3d18f7f343fcb00f/terraform/01_provider.tf)  with the constraints you set in this file